books/RayTracingInOneWeekend.html

```diff
@@ -66,7 +66,7 @@
 Graphics_ by Steve Marschner and Peter Shirley, or _Computer Graphics: Principles and Practice_
 by J.D. Foley and Andy Van Dam.
 
-Peter maintains a site related to this book series at https://in1weekend.blogspot.com/, which
+Peter maintains a site related to this book series at <a>https://in1weekend.blogspot.com/</a>, which
```

Comment:
Markdeep probably fixes this up automatically. The proper HTML tag would be

    <a href="https://in1weekend.blogspot.com/">https://in1weekend.blogspot.com/</a>

Reply:
Oh cool, glad you made upstream changes.  Didn't do the full HTML link since I wanted to keep it a small change. Appreciate you look at it!